students/biegon_piotr/lesson_03_functions/exponentiation.py

```diff
@@ -0,0 +1,14 @@
+print("Exponentiation\n")
+
+
+def power(a, n):
+    if n == 0:
+        return 1
+    if n >= 1:
```

Comment:
I think here `elif` would suit better here.
There so no way that both conditions gonna ever apply, so to empathize that `elif` is more preferred.